cognite/powerops/custom_modules/power_model_v1/data_models/views/function_inputs/BenchmarkingCollectorInput.view.yaml

```diff
@@ -27,15 +27,29 @@ implements:
     type: view
 version: "{{version}}"
 properties:
-  shopObjectiveValue:
-    container:
+  shopResults:
+    type:
+      space: "{{power_ops_type_space}}"
+      externalId: ShopResults
+    source:
       space: "{{power_ops_models}}"
-      externalId: FunctionData
-      type: container
-    containerPropertyIdentifier: direct1
-    name: shopObjectiveValue
+      externalId: ShopResult
+      version: "{{version}}"
+      type: view
+    direction: outwards
+    name: shopResults
+    description: An array of shop results.
+    connectionType: multi_edge_connection
+  shopRunInputs:
+    type:
+      space: "{{power_ops_type_space}}"
+      externalId: ShopRunInputs
     source:
       space: "{{power_ops_models}}"
-      externalId: ShopObjectiveValue
+      externalId: BenchmarkingShopRunInputDayAhead
```

Comment:
**question:** I'm not sure if this makes sense to link to as it's another function input type, will need to revisit when bluefield is up again to see it in the UI

Reply:
this is fine if we switch to making it a parallel to `ShopTriggerOutput` as mentioned in my other comment

Reply:
ShopTriggerOutput has got one shop result whereas BenchmarkingCollectorInput has got a list of all benchmarking shop results. I would leave BenchmarkingCollectorInput as it is. Would it not be fine as it is without parallelising to ShopTriggerOutput?